docs/data/joy/main-features/color-inversion/color-inversion.md

```diff
@@ -41,88 +36,52 @@ The color inversion feature is only available for `soft` and `solid` variants be
 - It doesn't work with browsers that don't support [CSS variables](https://caniuse.com/css-variables).
 
 ## Usage
 
-### Surface components
+### Supported components
 
-Surface components, including the Alert, Card, Drawer, ModalDialog, Menu, and Sheet, have the `invertedColors` prop to enable color inversion for `solid` and `soft` variants.
+All the surface components, including others like the Alert, Card, Drawer, Modal Dialog, Menu, and Sheet, accept the `invertedColors` prop for their `solid` and `soft` variants.
 
 {{"demo": "ColorInversionSurface.js"}}
 
 ### Portal popup
 
-By default, color inversion has no effect on the popup slot of the Autocomplete, Menu, and Tooltip components.
-To enable it, set `disablePortal` to true on the `slotProps`.
-
-:::info
-The popup slot of the Select component has `disablePortal` set to true by default.
-:::
+Color inversion does not affect the popup slot of the Autocomplete, Menu, and Tooltip components by default.
+To enable it, on the `slotProps` prop of these components, set `disablePortal` to true.
 
 {{"demo": "ColorInversionPopup.js"}}
 
-### Skip color inversion on a child
-
-To skip color inversion on a specific child, set `data-skip-inverted-colors` attribute to the component.
+### Turning it off on a specific child
 
-The component with `data-skip-inverted-colors` and its children will be excluded from the color inversion.
+Add the `data-skip-inverted-colors` attribute to the child component you want to turn color inversion off within an inverted container.
 
 {{"demo": "ColorInversionSkip.js"}}
 
 ### Apply color inversion to any parent
 
-Use `applySolidInversion` or `applySoftInversion` utilities to apply color inversion to any parent component.
-
-They are used internally by the surface components, e.g. [Card](/joy-ui/react-card/#inverted-colors), when `invertedColors` prop is set to true.
-
 ```js
 import { applySolidInversion, applySoftInversion } from '@mui/joy/colorInversion';
 ```
 
-Example usage for `sx` prop and `styled` API:
-
-- `sx` prop
-
-  ```js
-  import { applySolidInversion } from '@mui/joy/colorInversion';
-
-  <Box
-    sx={[
-      (theme) => ({
-        display: 'flex',
-        alignItems: 'center',
-        background: theme.vars.palette.neutral[900],
-      }),
-      applySolidInversion('neutral'),
-    ]}
-  >
-    …
-  </Box>;
-  ```
-
-- `styled` API
-
-  ```js
-  import { styled } from '@mui/joy/styles';
-  import { applySoftInversion } from '@mui/joy/colorInversion';
-
-  const StyledBox = styled(Box)(
-    ({ theme }) => ({
-      display: 'flex',
-      alignItems: 'center',
-      ...theme.variants.soft.primary,
-    }),
-    applySoftInversion('primary'),
-  );
-  ```
+Use the `applySolidInversion` or `applySoftInversion` utilities to add color inversion to any parent component.
+The Card component, for example, already uses it internally if containing the `invertedColors` prop.
```

Comment:
When skimming, it wasn't clear to me at first why the Card component was being mentioned here

```suggestion
(This is what the supported components use behind the scenes when the `invertedColors` prop is applied.)
```